src/patterns/molecules/error/error.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../../shared-styles/normalize"; // Always used first
+@use "../../../shared-styles/baseline";
+@use "../../../shared-styles/grid";
+@use "../../../shared-styles/media-query";
+@use "../../../shared-styles/rem";
+@use "../../../shared-styles/scale";
+@use "../../../shared-styles/size";
+@use "../../../shared-styles/space";
+@use "../../../shared-styles/typography";
+
+.error {
+  text-align: center;
+  @include size.max-inline-size(grid.$max-inline-size);
+  @include space.margin(auto, inline);
+  @include space.padding(grid.$column_gap, inline);
+  @include space.block($start: rem.calculate(baseline.for-units(12)), $end: rem.calculate(baseline.for-units(12)));
+  @include media-query.mq($from: medium) {
```

Comment:
(Out of scope.) While I'm on a roll with the naming revisionism... 😅  I don't much like `@include media-query.mq()` either. `@include media.query()` would read better, but not sure how feasible this is with how we're using the `mq` library. Will have a play and see if can get something sensible from it (into a separate PR).